test/requests/is_api_request_test.rb

```diff
@@ -20,7 +20,9 @@ def app
   test "test upload via web interface" do
     filename = GemFactory.gem_file(:example)
     header "Accept", "text/html"
-    post '/upload', { file: Rack::Test::UploadedFile.new(filename, 'application/octet-stream', true) }
+    silence_stream($stdout) do
+      post '/upload', { file: Rack::Test::UploadedFile.new(filename, 'application/octet-stream', true) }
+    end
```

Comment:
I could not see advantages by suppressing the output here... (`silence` in `setup` makes sense to me though...)

Reply:
Same as above.